radars/SRE.csv

```diff
@@ -0,0 +1,951 @@
+<h3>Pros</h3>
+<ul>
+<li>Easy to set up</li>
+<li>Easier to stay up to date with k8s releases through managed nodes</li>
+<li>Don’t have to worry about the control plane (master nodes)</li>
+<li>More strictly adheres to best security practices</li>
+<li>Integrated IAM support (though can be added to home spun clusters)</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Tied to AWS</li>
+<li>Less ability to customize</li>
+</ul>
+<h3>Trial Goal</h3>
+<p>The goal of trialing EKS is see if we can become more efficient and stay up to date without manually upgrading the cluster constantly with added security.</p>
+<p>When decision made: 5/1/2021</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+Jenkins,CI/CD,Hold,leaning,"<p><a href=""https://jenkins.io/"">Jenkins</a> is a build system. Built by the open source community it provides all the functionality needed for continuous integration.</p>
+<h3>Pros</h3>
+<ul>
+<li>Innumerable plugins</li>
+<li>Not the easiest interface to work with</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Sometimes low quality plugins due to open source community</li>
+</ul>
+<p>When decision made: 5/1/2021</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+Teamcity,CI/CD,Hold,leaning,"<p><a href=""https://www.jetbrains.com/teamcity/"">Teamcity</a> is a build system. Built by the folks at Jetbrains it provides all the functionality needed for continuous integration.</p>
+<h3>Pros</h3>
+<ul>
+<li>Solid interface</li>
+<li>Build templating</li>
+<li>Well built plugins built by the Teamcity team</li>
+<li>Great cloud build agent support on Kubernetes or using AWS AMIs</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Not as popular as Jenkins and so there are less plugins</li>
+<li>Costs money</li>
+</ul>
+<p>When decision made: 5/1/2020</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+Spinnaker,CI/CD,Invest,leaning,"<p><a href=""https://www.spinnaker.io/"">Spinnaker</a> is a continuous delivery tool that can work with many different types of infrastructure. While Spinnaker can work with anything we have chosen to use it primarily to manage the deployment pipelines for the applications that reside on our Kubernetes cluster.</p>
+<h3>Pros</h3>
+<ul>
+<li>Visual workflow for releases</li>
+<li>Very customizable</li>
+<li>Constantly evolving</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Very resource heavy application</li>
+<li>Difficult configuration</li>
+<li>Occasional there are significant changes required to upgrade</li>
+<li>Not super well documented</li>
+</ul>
+<p>When decision made: 3/5/2020<br />
+When decision will be revisited: 01/01/2022</p>
+"
+Concourse CI,CI/CD,Trial,budding,"<p><a href=""https://concourse-ci.org/"">Concourse CI</a> is an open-source continuous thing-doer. Built on the simple mechanics of resources, tasks, and jobs, Concourse presents a general approach to automation that makes it great for CI/CD. Could potentially replace <a href="""">Spinnaker</a> for CI.</p>
```

Comment:
Concourse seems far beyond the "Trial" stage for us

Reply:
Upgraded to invest